users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_sort_layer2.py

```diff
@@ -46,6 +44,22 @@ def sort_filters_by_center_freq(x):
     # move to log domain, not needed for center frequencies
     # f_resp = 20 * tf.math.log(f_resp) / tf.math.log(tf.constant(10.0))
 
+    return f_resp
+
+
+def sort_filters_by_center_freq(x):
```

Comment:
Please leave the old function unchanged. If this is because of black, maybe just add the new function in `specaug_configurable.py`. I sometimes hashed the function contents and this would break them (maybe not here, but better be safe).

Reply:
You marked this as resolved, but it is not. There are still changes in the old function.

Reply:
There is still a line removed.